README.md

```diff
@@ -2,55 +2,60 @@
 
 # Project Name
 
-> Description the project.
+Admin-dashboard
 
+> Description the project.
+> This is an admin-dashboard project use to test my knowledge and understanding of Grid in CSS. Though i also use some idea on Flexbox.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS ( Grid and Flexbox)
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+You will need the following on you machine to reproduce this project
+
+- Code editor preferebly Visual Studio Code (VScode).
+- Basic understanding of HTML, CSS Javascript languages
+
 ### Setup
 
+To clone this repository, please follow the steps below:
+
+- In the File menu, click Clone Repository.
+- Click the tab that corresponds to the location of the repository you want to clone. ...
+- Choose the repository you want to clone from the list.
+- Click Choose... and navigate to a local path where you want to clone the repository.
+- Click Clone.
+
 ### Install
 
+- VSCode.
```

Comment:
Weldone. But I notice that you choice of colors is not perfect. I prefer you stick to the colors provided by the template design for this project. 

### Your work
![image](https://user-images.githubusercontent.com/84156714/197339316-148070be-5bad-4e29-aa97-17de4945ec65.png)

### The design template
![image](https://user-images.githubusercontent.com/84156714/197339350-5e9e40c8-ed7f-45ee-9bac-3fdc22abbef9.png)